apps/demos/package.json

```diff
@@ -5,7 +5,8 @@
   "main": "",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "serve": "bos-loader -w -c"
+    "serve": "bos-loader -w -c",
+    "dev": "pnpm serve"
```

Comment:
let's not make this `dev`, I don't think we need to always run `bos-loader` when working on the engine so it's better to invoke it separately

Reply:
I end up needing it for anything but `sandbox`, which is most of my work. Any objection to adding a `dev:web` script that runs both?

Reply:
not at all, add any commands you want

It sounds like my comment underestimated the place of `bos-loader` in our workflow. Feel free to move forward with attaching it to `dev` if you feel thats the most intuitive setup for the team

Reply:
Well just _my_ workflow, but I won't be the typical user 😛  I was more concerned that maybe there's something about `bos-loader` that shouldn't be run as an NPM script alongside the JS.

I'll leave it out of the `dev` and add a new script in a separate PR. If we intend for sandbox and local to be mutually exclusive, then separate scripts for each would be a good way to make that distinction.